projects/go-lib/src/lib/components/go-icon-button/go-icon-button.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@tangoe/gosheets/base/variables';
+@import '~@tangoe/gosheets/base/mixins';
+
+.go-icon-button {
+  @include transition(background-color);
+
+  border: 0;
+  border-radius: $global-radius--round;
+  color: $base-dark;
+  cursor: pointer;
+  display: inline-flex;
+  outline-color: $theme-light-border;
```

Comment:
Not sure about this, would like to see what this looks like. We have a lot more control with a `box-shadow` so I would probably opt for that. Another question along these same lines is the interactivity of the button. Hover, focus, active states.

Reply:
Yeah, we should switch to using a `box-shadow` for this.